test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -3708,7 +3708,7 @@ public void Defect_MAGN_3212()
 
                     //Check the CBN for input/output ports
                     Assert.AreNotEqual(ElementState.Error, cbn.State);
-                    Assert.AreEqual(1, cbn.OutPorts.Count);
+                    Assert.AreEqual(0, cbn.OutPorts.Count);
```

Comment:
Was this test doing an assertion that was incorrect and is now fixed by the change here?

Reply:
Correct. Now that we remove the output ports for a function def, the count was updated to zero for this particular test.